### DISCUSSION TITLE
Welcome to socket.io Discussions!

### DISCUSSION THREAD

**Discussion by zishang520** [zishang520/socket.io#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by justvisiting** [comment]
Does this only implement socketio v4 server only or client as well?

**Reply by zishang520** [reply]
Currently there is only the server side, the client side is in the next plan.
